github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+
+-   The State Label will be changed to 'investigating or understood'
+
+-   The Area Label will be added
+
+-   The Milestone label will be added (As needed.)
+
+-   The Priority Label will be supplied (As needed.)
+
+> ****Defects and Doc issues****
+>
+> ***'Optional' Phases will apply to issues that are immediately understood allowing them to bypass many of the phases, moving the bug right to 'in-progress'.***
+
+****Enhancement Issues****
+
+> ***Where at 'review' the feature is reviewed for release and obtains any special sign-off needed from feature teams (if applicable). If not done (completed or has issues), go back to investigate (if needed) or move to design or in-progress for additional work. Repeat as needed. Once the feature can pass the review process (Whatever that may be for the given feature) it moves to closed***
+>
+> **Important! The Bug or Enhancement will remain un-assigned until someone is ready to pick up the work!** We do not want to just assign items to people who have work already since it is possible others may want to contribute to the fix/feature.
+>
+> At the very least the issue can be assigned a future Milestone if it is understood that is where the work needs to be done **OR** if there is not a clear path for the bug, leaving the Milestone blank functions as a backlog for issues without a release assigned. Once it reaches the current Milestone, the additional labels may be added.
+>
+> **Responsibility of the Area Owner**
+>
+> At this time, the Bug or Enhancement is **NOT** assigned to a specific individual, just the Area. Area owners need to be aware of the items in their queue and need to be aware of the Milestone so that they can review the active, unassigned items and be prepared to assign them once a resource is ready for work.
+>
+> Once the issue is ready to be worked on (either by resource of by Milestone deliverable) an individual will be assigned to do the work. The 'in-progress' label is applied now removing the 'understood' label').
+>
+> Work on the issue will continue until completed. i.e. the fix/feature is validated and approved by the Dev Lead and no regressions are associated with the issue. Once work is done and all necessary related tasks are completed, the item is closed.
+>
+> Specific build numbers will be used to update the Milestone label so we know specifically what build number shipped and what fixes are associated with it. Changing the label changes ALL the tagged issues at one time.
+>
+> Documentation can be taken from GitHub based on build number in the Milestone and consolidated for Release Notes.
+>
+> Once all work is completed and signed off on, we can RTW and close-down the Milestone and move any remaining items into the appropriate/next Milestone.
+
+Work Flow Process Table (Same as above - fewer words)
```

Comment:
If these say the same thing, lets use the one with fewer words.  That makes us have less copies to maintain and update as we change.